src/charm.py

```diff
@@ -238,6 +245,16 @@ def _on_storage_detaching(self, event: StorageDetachingEvent) -> None:
 
     def _on_install(self, _) -> None:
         """Handler for `install` event."""
+        sysctl_yaml = yaml.safe_load(Path("./src/templates/sysctl.yaml").read_text())
+        sysctl = SysctlConfig(config_params=sysctl_yaml["testing"], app_name=CHARM_KEY)
+        try:
+            sysctl.update()
+        except SysctlPermissionError as e:
+            logger.error(f"Error setting values on sysctl: {e.message}")
+            self._set_status(Status.SYSCONF_NOT_POSSIBLE)
+        except SysctlError:
+            logger.error("Error on sysctl")
```

Comment:
Should we handle somehow this error or re-raise it?

Reply:
For the kafka operator there is a health check that will see values for sysctl configs, so the charm will notice config not being optimal at one point.